NebulaModel/MultiplayerOptions.cs

```diff
@@ -7,6 +7,21 @@ namespace NebulaModel
     [System.Serializable]
     public class MultiplayerOptions : ICloneable
     {
+        [DisplayName("Nickname")]
+        public string Nickname { get; set; } = "Player";
+
+        [DisplayName("Color Red")]
+        [UIRange(0f, 1f)]
+        public float ColorR { get; set; } = 1f;
+
+        [DisplayName("Color Green")]
+        [UIRange(0f, 1f)]
+        public float ColorG { get; set; } = 0.6846404f;
+
+        [DisplayName("Color Blue")]
+        [UIRange(0f, 1f)]
+        public float ColorB { get; set; } = 0.24313718f;
```

Comment:
We should use the name `Mecha Color Red` to be more explicit

Reply:
> We should use the name `Mecha Color Red` to be more explicit

Should I reflect this in the variable name as well?

Reply:
yes please

Reply:
> yes please

Done in 58edede